experiments/browser_based_querying/src/TrustfallPlayground.tsx

```diff
@@ -274,6 +276,14 @@ export default function TrustfallPlayground(props: TrustfallPlaygroundProps): JS
             <div ref={resultsEditorRef} css={cssEditor} />
           </Paper>
         </Grid>
+        <Grid container item xs={4} direction="column" sx={{ flexWrap: 'nowrap' }}>
+          <Typography variant="h6" component="div">
+            Documentation Explorer
+          </Typography>
+          <Box sx={{ maxHeight: '85vh', overflowY: 'overlay', overflowX: 'hidden', mt: 2 }}>
```

Comment:
We might want to set the height explicitly here so it takes up a consistent amount of space no matter what the contents are.